src/Filament/Resources/ExceptionLogResource.php

```diff
@@ -31,14 +35,35 @@ public static function table(Table $table): Table
         return $table
             ->columns([
                 ExceptionColumn::make('exceptions'),
-                TextColumn::make('exception_logs_count')->counts('exceptionLogs')->label('Events'),
+                TextColumn::make('Events')
+                    ->getStateUsing(function (Model $record) {
+                        return $record->exceptionLogs()
+                            ->where('status', self::$statusFilter)
+                            ->count();
+                    }),
                 TextColumn::make('first_seen')->dateTime()->sortable(),
                 TextColumn::make('last_seen')->dateTime()->sortable(),
             ])
             ->defaultSort('last_seen', 'desc')
             ->filters([
                 SelectFilter::make('sites')
                     ->relationship('site', 'name'),
+                SelectFilter::make('status')
+                    ->default('unresolved')
+                    ->options([
+                        'unresolved' => 'Unresolved',
+                        'reviewed' => 'Reviewed',
+                        'ignored' => 'Ignored',
+                        'resolved' => 'Resolved',
```

Comment:
Here you can use the enum to get the key value.

```suggestion
            ExceptionLogStatus::UNRESOLVED->value => 'Unresolved',
            ExceptionLogStatus::RESOLVED->value => 'Resolved',
            ExceptionLogStatus::IGNORED->value => 'Ignored',
            ExceptionLogStatus::REVIEWED->value => 'Reviewed',
```